qa/workunits/rbd/cli_migration.sh

```diff
@@ -69,6 +76,22 @@ remove_images() {
     done
 }
 
+remove_namespace() {
+    local namespace=$1
+    local images=$(rbd ls --namespace ${namespace})
+    for IMAGE in ${images}; do
+        remove_image rbd/${namespace}/${IMAGE} || true
```

Comment:
`remove_images()` works on a hard-coded list of image names, I'd suggest to do the same here:
```suggestion
    for image in ${IMAGES}; do
        remove_image rbd/${namespace}/${image} || true
```

Reply:
But all the list of images from `${IMAGES}` may not belongs to that `namespace1` or `namespace2`
hence I listing the images inside namespace then removing it.

Reply:
That is fine -- this is a cleanup routine which is explicitly written to not fail if an image isn't there, or if it's there but not being migrated, etc.  Note the use of `|| true`.

Reply:
ok got it thanks, now made the changes accordingly.